lib/pkgtailor.js

```diff
@@ -60,30 +59,6 @@ const cFsConstants = (fs.constants || fs);
  */
 const c32BitPathPattern = /((\/lib)|(\/linux-x86-32))\/[^\/]+$/;
```

Comment:
maybe use `new RegExp("/(lib|linux-x86-32)/[^/]+$")` here? I have the feeling less quoting is easier to read.